test/examples/test_examples.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+
+# Copyright (c) 2002-2017 "Neo Technology,"
+# Network Engine for Objects in Lund AB [http://neotechnology.com]
+#
+# This file is part of Neo4j.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from test.integration.tools import IntegrationTestCase
+
+class ExamplesTest(IntegrationTestCase):
+
+    def setUp(self):
+        self.clean()
+
+    def test_autocommit_transaction_example(self):
+        from autocommit_transaction_example import AutocommitTransactionExample
+
+        example = AutocommitTransactionExample(self.bolt_uri, self.user, self.password)
+        example.add_person('Alice')
+
+        self.assertTrue(self.person_count('Alice') > 0)
+
+    def test_basic_auth_example(self):
+        from basic_auth_example import BasicAuthExample
+
+        example = BasicAuthExample(self.bolt_uri, self.user, self.password)
+
+        self.assertTrue(example.can_connect())
+
+    # def test_config_connection_timeout_example(self):
+    #     from config_connection_timeout_example import ConfigConnectionTimeoutExample
+    #     pass
+        
+    # def test_config_max_retry_time_example(self):
+    #     from config_max_retry_time_example import ConfigMaxRetryTimeExample
+    #     pass
+
+    # def test_config_trust_example(self):
+    #     from config_trust_example import ConfigTrustExample
+    #     pass
```

Comment:
These commented out ones, I'm not entirely sure they have a mapping to the Python driver.